var/spack/repos/builtin/packages/dftbplus/package.py

```diff
@@ -0,0 +1,145 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+import os
+
+
+class Dftbplus(MakefilePackage):
+    """DFTB+ is an implementation of the
+    Density Functional based Tight Binding (DFTB) method,
+    containing many extensions to the original method."""
+
+    homepage = "https://www.dftbplus.org"
+    url      = "https://www.dftbplus.org/fileadmin/DFTBPLUS/public/dftbplus/19.1/dftbplus-19.1.tar.xz"
```

Comment:
I think the way this URL is, `spack versions dftbplus` won't find any additional versions of the package. If you add:
```python
list_url = "https://www.dftbplus.org/fileadmin/DFTBPLUS/public/dftbplus/"
list_depth = 1
```
it should fix the problem.